### ISSUE TITLE
Support `rotate` prop also for root Svg component

### ISSUE THREAD

**Issue by jukben** [react-native-community/react-native-svg#533]
Hey, 

currently is not supported to do. I don't know if it's a bug or not but it would be nice to have the same behavior as on the web: `<svg transform="rotate(270)"...>`

**Current behavior**
Rotate prop on Svg component is ignored

**Expected behavior**
Rotate prop on Svg component will rotate svg

**Example of usage:**

```js
 <Svg
                height="100"
                width="100"
                rotate="270"
            >
                <Circle
                    cx="50"
                    cy="50"
                    r="45"
                    stroke="blue"
                    strokeWidth="2.5"
                    fill="green"
                />
</Svg>
```

What do you think?

**Comment by sms1989** [comment]
I think issue #475 can help you

**Comment by dustinsavery** [comment]
@jukben this is solved with #507. This is currently in `react-native-svg@6.0.1-rc.1`.
